config/sidekiq.yml

```
@@ -4,6 +4,7 @@
 :logfile: ./log/sidekiq.json.log
 :timeout: 4
 :queues:
+  - [delivery_transactional, 7]
```

```suggestion
  - delivery_transactional
```

I'm not sure whether this is a good idea, but if we did this, it would mean that Sidekiq _only_ takes jobs from the `delivery_transactional` queue until it's empty. I wonder if this is better for getting the emails out straight away, but it does mean if the queue gets long, other emails are delayed... not sure...

---

Yeah, I was thinking about this - I think its worth trying the weighted method first. I'd expect to see a good improvement anyway because the emails will be at the start of a queue, instead of behind potentially 100,000s of other emails. If this didn't work, I'd be tempted to try a dedicated Sidekiq process for that queue instead make it a blocking queue.